build-system/global-configs/README.md

```diff
@@ -10,5 +10,5 @@ besides 1 or 0.
 - `name`: Experiment name
 - `command`: Command used to build the experiment
 - `issue`: The issue tracker URL for this experiment
-- `expirationDateUTC`: The experiment expiration date in YYYY-MM-DD format, in UTC
+- `expirationDateUTC`: The experiment expiration date in YYYY-MM-DD format, in UTC. If an experiment is expired, it will fail the build process. There is currently no realtime check for the expiration time.
```

Comment:
Thanks for clarifying the expiration date. 
Realtime check will be difficult to add and can cause a lot of confusion. I don't believe we will be adding that in the future. Do you think we should rephrase the sentence to not mention the concept? Instead we can explain how the expiration data affects the build process, and note that the experiment will be started and ended the same time along with our release process.